Issutracker : #iXXXXX#
CWS         : N/A
Author      : NAKATA Maho <maho@openoffice.org> (JCA)
Description : Do not make a link as /usr/bin/soffice

Index: scp/source/office/unxbasic.scp
===================================================================
RCS file: /cvs/installation/scp/source/office/Attic/unxbasic.scp,v
retrieving revision 1.49.8.9
diff -u -r1.49.8.9 unxbasic.scp
--- scp/source/office/unxbasic.scp	27 Aug 2004 11:43:50 -0000	1.49.8.9
+++ scp/source/office/unxbasic.scp	15 Jan 2005 10:22:08 -0000
@@ -924,12 +924,6 @@
 		Dim sofficeFile As String
 		Dim linkName As String
 
-		sofficeFile = DirEntry(GetSetupEnv.DestPath, "program", "soffice")
-
-		linkName = "/usr/bin/soffice"
-
-		shell("/bin/sh -c " & "'ln -s " & chr(34) & sofficeFile & chr(34) & " " & linkName & " >/dev/null 2>&1'")
-
 	End Sub
 
 };
